packages/types/src/diagnostics.ts

```diff
@@ -1,6 +1,6 @@
 import type { GasPrices } from './gas_prices';
 
-export interface Diagnostics {
+export type Diagnostics = {
```

Comment:
Why do you switch from interface to type here? Is it because we don't plan on ever extending the Diagnostics? https://blog.logrocket.com/types-vs-interfaces-in-typescript/

Reply:
Interfaces don't map to `Record<string, unknown>` which was forced by typescript-eslint. I **really** despise typescript-eslint and plan to drop it, but this was the workaround for now.